boxoffice/templates/boxoffice.js

```diff
@@ -254,7 +254,7 @@ $(function() {
                 }
               });
 
-              if(valid_discount_coupon) {
+              if (valid_discount_coupon) {
                 boxoffice.ractive.set('order.line_items',line_items);
```

Comment:
Not necessarily, no? Since the line items that have a pre-applied discount will be changed anyway. I don't think `valid_discount_coupon` is required at all. Also, `valid_discount_coupon` as a variable name is slightly mis-leading, since it actually is just set if a line item has a discount applied to it.